10 lessons (#193)
xmartlabs/blog: _posts/2022-03-03-10-years-in-tech-industry.md

Xmartlabs began ten years ago under similar circumstances. Several factors influenced our decision: cloud computing lowered costs, the lean startup methodology was rising in popularity, globalization and massive adoption of high-speed internet, mobile as a new tech platform, and a pool of undiscovered talent in Uruguay. Ultimately though, we were young and had nothing to lose, so we went for it.

These past weeks of insight that inevitably come with each poignant date, such as birthdays and anniversaries, led us to come up with some lessons learned during the past decade. With all honesty, when we set out to do this back in 2012, we never thought we'd make it this far. So if anything, we're not trying to give out advice from a high horse, but a place of humility and appreciation. With any luck, you too will find some of them helpful :)

These are the lessons that stand out the most as we look back at our 10-year journey.

## 1. Work Among Friends

Since the beginning, the company needed to be people and relationship-oriented. It was not about a business transaction but an interpersonal relationship. And this approach had a lot to do with why we set out to build our own company.

> Although today it might be the norm, back in 2012, Uruguayan companies weren't that much people-oriented. We felt there was a lot to do in the employment sector. At the time, it was not flexible in terms of hours or suspicion of trust, responsibility, and autonomy. With Xmartlabs, we wanted to change that.

Reviewer: ```suggestion
Although today it might be the norm, back in 2012, Uruguayan companies weren't that much people-oriented. We felt there was a lot to do in the employment sector. At the time, it was not flexible in terms of hours, which also touched on subjects like trust, responsibility, and autonomy. With Xmartlabs, we wanted to change that.
```

> Although today it might be the norm, back in 2012, Uruguayan companies weren't that much people-oriented. We felt there was a lot to do in the employment sector. At the time, it was not flexible in terms of hours or suspicion of trust, responsibility, and autonomy. With Xmartlabs, we wanted to change that.

Reviewer: ```suggestion
Although today it might be the norm, back in 2012, Uruguayan companies weren't that much people-oriented. We felt there was a lot to do in the employment sector. At the time, it was not flexible in terms of hours and many times didn't rely on trust, responsibility, and autonomy. With Xmartlabs, we wanted to change that.
```

> Staying close to your team, listening and being with them, not asking for things you wouldn't do; are all key to creating a horizontal work environment. Being intentional about it and bringing it up constantly, so it was on the team's top of mind made it easier to maintain a good working environment.

Reviewer: ```suggestion
Staying close to your team, listening and being with them, not asking for things you wouldn't do; are all key to creating a horizontal work environment. Being intentional about it and bringing it up constantly, so it was on the team's top of mind, made it easier to maintain a good working environment.
```

This approach made Xmartlabs what we believe is a great place to work. Our turnover rate pre-pandemic was less than 5-1% (on an industry that then had a 13+% turnover rate). This is also reflected in the longevity of some of our team. Many of the old Xmartlabers themselves recommend new people for us to hire.

<figure>
  <img style="width: 40%" src="/images/10-lessons-in-tech-industry/xl-working-with-friends.jpg" alt="working among friends">
</figure>

## 2. First team members set the company culture

> We always aim to create the best innovative team in South America, but we never compromise the team's human qualities. 

Reviewer: ```suggestion
We always aimed to create the best innovative team in South America, but we never compromised our team's human qualities. 
```

> Important things are accomplished by the team and not by an individual. Following that thought, we began to recruit (1 to 1) talented acquaintances and friends in the industry who also shared our values and principles. In this way, they would support Xmartlabs' vision and help lay the foundations of the culture we wanted in Xmartlabs.

Reviewer: ```suggestion
Milestones are accomplished by teams and not by individuals on their own. With that in mind, we began recruiting (1 to 1) talented acquaintances and friends in the industry who shared our values and principles. This way we hoped they'd support Xmartlabs' vision and help lay the foundations of the culture we wanted in Xmartlabs.
```

Different key roles were essential to put the company on its feet during those early stages:

> - A designer that pulió/dusted off our services (especially on our website) and added value and differentiation.

Reviewer: ```suggestion
- A designer that dusted off our services (especially on our website) and added value and differentiation.
```

- Accounting roles helped us organize our finance and allowed us to leverage our growth without external investment.

> - HR people that helped us shape the recruiting and internal process.

Reviewer: ```suggestion
- HR specialists that helped us shape the recruiting and internal process.
```

> However, we did have to cover a considerable amount of roles simultaneously. On many of them, we had no background whatsoever (legal, marketing, sales, business management), which presented a huge opportunity to learn on the go.

Reviewer: ```suggestion
However, we did have to cover a considerable amount of roles simultaneously. We had no background whatsoever in legal, marketing, sales, or business management, which presented a huge opportunity for learning on the go.
```

## 3. Learn From Your Clients

Anyone who started a company knows how much first clients teach you and how valuable those lessons are. They especially point you in the right direction by helping you understand what they value, pain points, and drivers they have. 

> Negative feedback it's also crucial as it will eventually help you improve. Those first reviews and bits of advice have an enormous impact on you (even if they're small tips like "B*uy noise-canceling headsets"*).

Reviewer: ```suggestion
Negative feedback it's also crucial as it will eventually help you improve. Those first reviews and bits of advice have an enormous impact on you (even if they're small tips like *"Buy noise-canceling headsets"*).
```

> Clients also helped us to adopt a *modus operandi* and a work ethic based on empathy and, just like with teammates, interpersonal bonds. They are also good to test yourself. In our case, we proved (to clients and ourselves) that we were up to any challenge and that being an offshore company was not a disadvantage or obstacle.

Reviewer: ```suggestion
Clients also helped us to adopt a *modus operandi* and a work ethic based on empathy and, just like with teammates, interpersonal bonds. They're also good to test yourself. In our case, we proved (to clients and ourselves) that we were up to any challenge and that being an offshore company was not a disadvantage or obstacle.
```

<figure>
  <img style="width: 100%" src="/images/10-lessons-in-tech-industry/first-website.png" alt="Xmartlabs’ website during those first years.">
  <figcaption>Xmartlabs’ website during those first years.</figcaption>
</figure>

## 7. We're never alone

Since our very beginning, we were made aware of the importance of weaving bonds with our surrounding community, this was mainly thanks to the accelerator we began in. Our first commercial mission and first move to an independent office was made together with other entrepreneurs we met there. 

We also got involved by giving several talks in main events, even when our team was small. OSS was also a big part of this because, although we didn’t have that many success stories, we had some type of value to give back to the community, and they were always open to receiving it.

## 8. Know who you don’t want to become

> We didn't have a completely clear vision of what we wanted to be. However, we did know what we didn't want to become: a colder, more transactional, and corporative business. Perhaps it's not the easiest way of carrying out a company because some things are left to luck, but it worked for us, especially at the outset. We always bet on organic growth and set up objectives based on the opportunities that came up or on aspects we need to improve.

Reviewer: ```suggestion
We didn't have a completely clear vision of what we wanted to be. However, we did know what we didn't want to become: a colder, more transactional, and corporative business. Perhaps it's not the easiest way of carrying out a company because some things are left to chance, but it worked for us, especially at the outset. We always bet on organic growth and set up objectives based on the opportunities that came up or on aspects we need to improve.
```
